.github/actions/publish_to_cf_pages_production/action.yml

```
@@ -0,0 +1,18 @@
+name: publish_to_cf_pages_production
+description: publish to cloudflare pages (production)
+inputs:
+  CLOUDFLARE_ACCOUNT_ID:
+    description: "Cloudflare account id"
+    required: true
+  CLOUDFLARE_API_TOKEN:
+    description: "Cloudflare token"
+    required: true
+runs:
+  using: composite
+  steps:
+    - name: Publish to cloudflare pages (production)
```

```suggestion
    - name: Publish to cloudflare pages (production)
      env:
          CLOUDFLARE_ACCOUNT_ID: ${{ inputs.CLOUDFLARE_ACCOUNT_ID }}
          CLOUDFLARE_API_TOKEN: ${{ inputs.CLOUDFLARE_API_TOKEN }}       
```
you need to pass the inputs as env to be pickable by wrangler

---

done